### PR TITLE
[MIG]Migrated web widget many2many_tags multi selection in v9

### PR DESCRIPTION
**Pull request by jas-odoo**
Migrated web widget many2many_tags multi selection in v9.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please sign the CLA: https://odoo-community.org/page/cla

**Comment by hbrunn** [comment]
:+1: provided CLA

**Comment by pedrobaeza** [comment]
:+1:

**Comment by pedrobaeza** [comment]
I have merged this thinking that Holger checked the CLA, but he meant that OK if you signed the CLA, so @jas-odoo, please sign the CLA.

File: web_widget_many2many_tags_multi_selection/__openerp__.py [hunk 1]
```diff
@@ -21,8 +21,8 @@
```
Comment (elicoidal) [inline web_widget_many2many_tags_multi_selection/__openerp__.py:21]: Can you take the opportunity to simplify headers following new templates?

File: web_widget_many2many_tags_multi_selection/README.rst [hunk 2]
```diff
@@ -29,6 +29,7 @@ Contributors
 ------------
 
 * Sylvain Calador <sylvain.calador@akretion.com>
+* Jamin Shah <jaiminshah2009@gmail.com>
```
Comment (elicoidal) [inline web_widget_many2many_tags_multi_selection/README.rst:32]: please update content using the newest README template.

File: web_widget_many2many_tags_multi_selection/__openerp__.py [hunk 3]
```diff
@@ -21,8 +21,8 @@
 
 {
     'name': 'Tags multiple selection',
-    'version': '8.0.0.1.0',
-    'author': 'Akretion, Odoo Community Association (OCA)',
+    'version': '9.0.1.1.0',
+    'author': 'Akretion, Odoo Community Association (OCA), Jamin Shah',
     'depends': [
         'web',
     ],
```
Comment (elicoidal) [inline web_widget_many2many_tags_multi_selection/__openerp__.py:28]: remove empty keys

**Comment by hbrunn** [comment]
Should we revert the merge commit? It could be a dangerous precedent to have non-cla-code in the main branch. And now there's still the opportunity to just reset to HEAD~1 and force pushing, we can't do that if new commits come on top of that

**Comment by legalsylvain** [comment]
:+1: for revert the merge.

**Comment by hbrunn** [comment]
@pedrobaeza could you do this?

**Comment by hbrunn** [comment]
@jas-odoo please resubmit after signing the CLA

**Comment by pedrobaeza** [comment]
Done and proposed again in https://github.com/OCA/web/pull/323

**Pull request merged**